keel-policy/src/main/kotlin/com/netflix/spinnaker/keel/policy/exceptions/MalformedMessageException.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.policy.exceptions
+
+class MalformedMessageException(
```

Comment:
Probably need a handler for this to set an appropriate HTTP status (400 looks like) if it causes a request to fail.